internal/backends/winit/event_loop.rs

```diff
@@ -565,6 +565,12 @@ pub fn run(quit_behavior: i_slint_core::backend::EventLoopQuitBehavior) {
     let mut winit_loop = not_running_loop_instance.instance;
     let clipboard = not_running_loop_instance.clipboard;
 
+    // When we need to apply window properties such as title or the width/height (due to constraints),
+    // we receive a custom event. The application implies querying the current window size, so do all of
+    // this after processing all other events, such as resize events: At MainEventsCleared and before
+    // rendering. This is to avoid the jitter as described in  https://github.com/slint-ui/slint/issues/1269
+    let mut windows_with_pending_property_updates = Vec::new();
```

Comment:
Perhaps we can reuse this to avoid a heap allocation on every call?

Reply:
That's right, that is why the Vec is placed here, so that it's not allocated every time winit calls our event handling closure.